util/client.js

```diff
@@ -63,3 +64,14 @@ export function tryPostLoginRedirect({ route }) {
   }
   return false;
 }
+
+const LIKECO_FRONTEND_MODE_COOKIE_KEY = 'likeco_frontend_mode';
+
+export function toggleFrontendMode() {
+  if (Cookies.get(LIKECO_FRONTEND_MODE_COOKIE_KEY)) {
+    Cookies.remove(LIKECO_FRONTEND_MODE_COOKIE_KEY);
+  } else {
+    Cookies.set(LIKECO_FRONTEND_MODE_COOKIE_KEY, 'legacy');
+  }
```

Comment:
we actually has `vue-cookie` as `this.$cookie`, do you want to use that or just use `js-cookie` ?

Reply:
Perhaps we use [tiny-cookie](https://www.npmjs.com/package/tiny-cookie), which depends on `vue-cookie`